opennms-webapp/src/main/webapp/admin/userGroupView/users/modifyUser.jsp

```diff
@@ -450,6 +453,29 @@
             <input class="form-control" type="text" id="homePhone" name="homePhone" value='<%=(homePhone == null ? "":homePhone)%>'/>
           </div>
         </div>
+
+    <div class="form-group">
+          <label for="timeZoneId" class="col-sm-2 control-label">Time Zone:</label>
+          <div class="col-sm-10">
+            <select class="form-control" id="timeZoneId" name="timeZoneId">
+<%
+              List<String> zones = new ArrayList<>();
+              zones.add("");
```

Comment:
I was unable to switch back to using the default time zone after selecting a specific time zone. Should this work currently?